pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -98,6 +98,7 @@ stdenv.mkDerivation rec {
     python3
     ruby
     glib # for gdbus-codegen
+  ] ++ optionals stdenv.isLinux [
```

Comment:
Could you please use qualified `stdenv.lib.optionals`? We really should get rid of the `with` statement.

Otherwise looks good to me if it builds on Darwin.

Reply:
Used `stdenv.lib.optionals` here, but left other `optionals` etc. unchanged.